tests/unit/resource/CMakeLists.txt

```diff
@@ -15,6 +17,8 @@ set(tests
     suspend_thread_timed
 )
 
+set(cross_pool_injectionr_PARAMETERS THREADS_PER_LOCALITY all)
```

Comment:
```suggestion
set(cross_pool_injection_PARAMETERS THREADS_PER_LOCALITY all)
```

Reply:
good spot. you must really read the code thoroughly to notice stuff like that. I'm impressed.